Lib/datetime.py

```diff
@@ -1025,15 +1025,82 @@ def __repr__(self):
 
 
     def ctime(self):
-        "Return ctime() style string."
+        """Return ctime() style string.
```

Comment:
Instead of the code example, I'd prefer adding "such as 'Fri Oct  5 00:00:00 2001'" to the end of the existing docstring.

Reply:
Will fix, although I'm curious what the reasoning is - does it make it more concise?

Reply:
Yes, more concise. And there aren't any other REPL examples in that module.